Releases.md

```diff
@@ -6,6 +6,34 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at
 https://github.com/denoland/deno_install
 
+### v0.32.0 / 2020.02.03
+
+- BREAKING CHANGE: Replace formatter for "deno fmt" use dprint (#3820, #3824,
+  #3842)
+- BREAKING CHANGE: Remove std/prettier (#3820)
+- BREAKING CHANGE: Remove std/installer (#3843)
+- BREAKING CHANGE: Remove --current-thread flag (#3830)
+- BREAKING CHANGE: Deno.makeTempDir() checks permissions (#3810)
+- feat: deno install in Rust (#3806)
+- feat: Improve support of type definitions (#3755)
+- feat: deno fetch supports --lock-write (#3787)
+- feat: deno eval supports --v8-flags=... (#3797)
+- feat: descriptive permission errors (#3808)
+- feat(std/node): Endianness (#3833)
+- feat(std/node): Partial os polyfill (#3821)
+- feat(std/examples): bring back xeval (#3822)
+- fix: Panic on cache miss (#3784)
+- refactor: Use tokio::main attribute in lib.rs (#3831)
+- refactor: NO_BUILD_SNAPSHOTS flag to speed up incremental builds (#3847)
+- refactor: Provide TS libraries for window and worker scope (#3771, #3812,
+  #3728)
+- refactor(deno_core): error tracking and scope passing (#3783)
+- refactor(deno_core): rename PinnedBuf to ZeroCopyBuf (#3782)
+- refactor(deno_core): change Loader trait (#3791)
+- upgrade: Rust 1.41.0 (#3838)
+- upgrade: rusty_v8 (#3816)
```

Comment:
rusty_v8 update is an internal change - unless V8 version changed?

Reply:
I took suggestion from previous release where there were several updates. I'm fine to remove 👍